collect.py

```diff
@@ -7,9 +8,10 @@
 from PIL import Image
 from selenium import webdriver
 from selenium.common.exceptions import NoAlertPresentException, NoSuchWindowException, TimeoutException
-
```

Comment:
Nit: Don't remove the newline here :)
As a rule, never remove newlines that aren't part of the changes you are making.